### PR TITLE
all: various cleanup

### PR DESCRIPTION
**Pull request by cedric-parisi**
*Replace testify/mock by golang/mock
*Remove brank.as/serviceutil as it is a private repository

### CODE REVIEW

File: services/accounts/accounts_test.go [hunk 1]
```diff
@@ -5,39 +5,44 @@ import (
...
 	accountspb "github.com/openbank/gunk/gunk/v1/accounts"
 	typespb "github.com/openbank/gunk/gunk/v1/types"
-	"github.com/openbank/openbank/storage"
+	"github.com/openbank/openbank/storage/mock_storage"
 )
 
 func TestHandler_GetAccount(t *testing.T) {
+	ctrl := gomock.NewController(t)
+	defer ctrl.Finish()
+	m := mock_storage.NewMockAccountStore(ctrl)
```
Comment (cuonglm) [inline services/accounts/accounts_test.go:20]: Just `NewAccountStore` is enough. We do have `mock_storage` prefix already.

File: services/accounts/accounts_test.go [hunk 2]
```diff
@@ -5,39 +5,44 @@ import (
...
+
 	type args struct {
-		ctx context.Context
 		req *accountspb.GetAccountRequest
 	}
 	tests := []struct {
 		name      string
 		args      args
-		mockCalls func(m *storage.MockAccountStore)
+		mockCalls func(m *mock_storage.MockAccountStore)
```
Comment (cuonglm) [inline services/accounts/accounts_test.go:28]: Same as above.

**Review by cuonglm — CHANGES_REQUESTED** [review]

**Comment by cuonglm** [comment]
@cedric-parisi LGTM, just some minor requests of naming.

**Review by mvdan — APPROVED** [review]
the -10k diff is glorious :)

File: services/accounts/accounts_test.go (hunk 1 above)
Comment (cedric-parisi) [reply services/accounts/accounts_test.go:20]: `mock_storage.go` is a generated file by `golang/mock` gen tool :)

**Pull request merged**